### PR TITLE
Morgue Updating [Hooks P4]

### PR DESCRIPTION
**Pull request by AffectedArc07**
**What does this PR do:**
This moves the `update_morgue()` proc directly into `mob/Login()` and `mob/Logout()` instead of going through a hook first.

Tested with another player (Thanks DoctorDrugs)

**Changelog:**
:cl: AffectedArc07
tweak: Morgue updates happen slightly better now
/:cl:

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
conflicts.

File: code/modules/mob/new_player/login.dm [hunk 1]
```diff
@@ -27,7 +27,7 @@
 		loc = pick(watch_locations)
 */
 
-	callHook("mob_login", list("client" = client, "mob" = src))
+	update_morgue() // Why is this called for new players that dont even have a morgue
```
Comment (Fox-McCloud) [inline code/modules/mob/new_player/login.dm]: Test if this can be removed, and, if so, do so, please.

Comment (AffectedArc07) [reply code/modules/mob/new_player/login.dm]: Will do

**Comment by Fox-McCloud** [comment]
conflicts.

**Pull request merged**